lib/utilities/aclUtils.js

```diff
@@ -117,7 +117,7 @@ aclUtils.parseAclXml = function parseAclXml(toBeParsed, log, next) {
 
         if (!result.AccessControlPolicy.Owner
```

Comment:
what about result.AccessControlPolicy.Owner[0].ID[0], do we need to check it is typeof 'string' and not just another object or array?